src/test/java/com/epam/esm/repository/user/UserRepositoryImplTest.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.esm.repository.user;
+
+import lombok.RequiredArgsConstructor;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+class UserRepositoryImplTest {
+  private final UserRepository repository;
+
+  @Test
+  @Transactional
+  void find_ShouldReturn_User_True() {
+    long expectedId = 4;
+    long actualId = repository.find(expectedId).get().getUserId();
```

Comment:
Can we get `NullPointerException`

Reply:
Fixed( add empty() case in RepositoryUser)